lib/client.ts

```diff
@@ -50,6 +49,21 @@ class Client {
         Client.connections.set(key, this);
     }
 
+    private __pingPongTimeout?: NodeJS.Timeout;
+    private async __pingPong() {
+        if (this.__pingPongTimeout) clearTimeout(this.__pingPongTimeout);
+        const reqId = await this.ping();
+        return this
+            .read(reqId)
+            .catch(() => this.reconnect())
+            .finally(() => {
+                this.__pingPongTimeout = setTimeout(() => {
```

Comment:
shouldn't this be setInterval? setTimout is executed only once, we want it executing every 20 seconds